docs/DeveloperGuide.md

```diff
@@ -260,6 +250,14 @@ From the diagram illustrated above:
 ### FindPool feature
 This feature allows users to find a pool that contains a passenger with a provided keyword in their name.
 
+Design considerations include the `findPool` command being able to be used in conjunction with the `pool` or `unpool`  command. Due to the limitations of the current iteration of GME, each passenger can only belong to one pool. As the number of passengers and pools increases, it may be hard to keep track of the passengers that exist in pools and those that dont, thus, the user might attempt to create a new pool with a passenger and be informed that that passenger already exists in a pool. The `findPool` feature thus allows the user to find the pool that the passenger already exists in to either confirm if they are in the correctly assigned pool, or `unpool` them if they are not. The `findPool` command works similarly to the `find` command, except that it currently only supports the use of the name prefix: "/n"
+
+The activity diagram below encapsulates the user workflow of attempting to pool a passenger, finding out that the passenger already exists in a pool, and using the `findPool` command to rectify the issue:
+
+![Activity Diagram for a user using Unpool](images/UnpoolActivityDiagram.png)
```

Comment:
unpool acitivity diagram is in findpool feature. perhaps rename the diagram so not to confuse future developers?
"Activity Diagram for a user using findpool with unpool"
maybe emphasize that the focus is on findpool here